packages/product-json-to-csv/src/map-product-data.js

```diff
@@ -50,6 +51,31 @@ export default class ProductMapping {
     this.languages = languages || [language]
     this.multiValDel = multiValueDelimiter
     this.createShortcuts = createShortcuts
+
+    this.productLevelProperties = [
+      'id',
+      'key',
+      'version',
+      'createdAt',
+      'lastModifiedAt',
```

Comment:
do we really need `createdAt` and `lastModifiedAt` here? Just thinking since we dont update them no?

maybe we could add a comment above with the link to where to fetch this list?

Reply:
all attributes which have the same name as any item from this list are prefixed with `attribute.` string. So I would say we should also prefix `createdAt` because then it can make problems when importing.

Reply:
updated here 4af192e